lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -619,7 +621,7 @@ def add_echeck(xml, check, options = {})
         add_bank_account_type(xml, check)
         xml.tag! :ECPAuthMethod, options[:auth_method] if options[:auth_method]
         xml.tag! :BankPmtDelv, options[:payment_delivery] || 'B'
-        xml.tag! :AVSname, (check&.name ? check.name[0..29] : nil) if get_address(options).blank?
+        xml.tag! :AVSname, (check&.name ? I18n.transliterate(check.name)[0..29] : nil) if get_address(options).blank?
```

Comment:
Can you also add a test for this?

Reply:
It would also be good to tests this remotely

Reply:
Done!